bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_3drepo.cpp

```diff
@@ -432,7 +462,9 @@ bool RepoModelImport::importModel(std::string filePath, uint8_t &err)
 		repoInfo << "Loading BIM file [VERSION: " << incomingVersion << "]";
 
 		size_t metaSize = REPO_VERSION_LENGTH + sizeof(fileMeta);
-		if (incomingVersion == REPO_V3)
+		//TODO: Clean up the version string/number/comparison stuff into a class
```

Comment:
I think either do it, or log a maintenance issue against it.

Inline todos just gets forgotten about and never get done 😆

Reply:
though a class for version comparison seems a bit over kill?

Reply:
also since you're no longer testing against the strings, you may as well remove the strings from the header and have 2 int depicting the min/max supported versions